src/MockClassifier.UnitTests/MockClassifier.UnitTests.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Buerokratt.Common" Version="1.0.53-prerelease" />
```

Comment:
Well - darn it I thought I'd fixed this reference... Maybe I didn't save or something?!  This is something to watch in future PRs from me...

Reply:
Yeah the package reference thing always catches me off guard too 😢